packages/analytics/dashboard-renderer/src/components/SimpleChartRenderer.vue

```diff
@@ -7,6 +7,7 @@
     <SimpleChart
       :chart-data="data"
       :chart-options="mappedChartOptions"
+      :height="`${props.height}px`"
```

Comment:
you don't use `props` in the template 
```suggestion
      :height="`${height}px`"
```